code/scripting/api/libs/ui.cpp

```diff
@@ -220,7 +220,7 @@ ADE_FUNC(isAutoselect, l_UserInterface_PilotSelect, nullptr,
          "Determines if the pilot selection screen should automatically select the default user.", "boolean",
          "true if autoselect is enabled, false otherwise")
 {
-	return ade_set_args(L, "b", Cmdline_benchmark_mode);
```

Comment:
Please also add an `ADE_VIRTVAR` that returns the command line value so that a script can replicate the behavior of the "normal" menu.